src/Microsoft.Health.Fhir.Core/Features/Operations/DataConvert/Models/GetTemplateSetFailedException.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.Health.Fhir.Core.Exceptions;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.DataConvert.Models
+{
+    public class GetTemplateSetFailedException : FhirException
```

Comment:
>GetTemplateSetFailedException [](start = 17, length = 29)

I'm having trouble with two verbs inside it.
Is it ` GetTempate` failed to be set? Or we try to get `TemplateSet` and fail?
If it's later can it be `TempateCollection` instead of `TemplateSet`

Reply:
Actually it means "template set". We can change to collection. Thanks

Reply:
You don't have to change it, as long as name of exception can be less ambiguous.

---
In reply to: [521420365](https://github.com/microsoft/fhir-server/pull/1434#discussion_r521420365) [](ancestors = 521420365)